### ISSUE TITLE
Accessibility: Potential issues with duplicate `aria-label` in reused components

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/web-stories-wp#11028]
## Task Description

Consider the following scenario:

<img width="1348" alt="Screenshot 2022-03-22 at 22 37 00" src="https://user-images.githubusercontent.com/841956/159580385-b44a9ea4-2def-48b4-99ed-cfaa892ebf54.png">

Here, the color picker for the text element is displayed twice, once in the sidebar and once in the floating toolbar. Both have the same `aria-label="Color input: Text color" attribute.

According to the [`landmark-unique`](https://dequeuniversity.com/rules/axe/4.3/landmark-unique?application=axe-puppeteer) aXe rule this violates a best practice which says those labels should be unique.

Example run: https://github.com/GoogleForCreators/web-stories-wp/runs/5647019338?check_suite_focus=true

We should look into this and perhaps either (selectively) disable this rule or find ways to make these labels unique but still keep them useful.

Any thoughts appreciated.

**Comment by BrittanyIRL** [comment]
(@felipebochehin87 there's details on what to test in the PR)

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using PR #11126 and comparing it with Main branch.

No error alert displayed when using axe DevTools (related to unique landmarks).

![moderate PR.png](https://images.zenhubusercontent.com/61d5f112aca19e2c5bb2441f/8192e7b4-7747-4128-93ac-2e324e37068e)
